src/object/wind.cpp

```diff
@@ -83,13 +90,22 @@ Wind::update(float dt_sec_)
   if (!blowing) return;
   if (m_col.m_bbox.get_width() <= 16 || m_col.m_bbox.get_height() <= 16) return;
 
-  // TODO: nicer, configurable particles for wind?
-  if (graphicsRandom.rand(0, 100) < 20) {
+  Vector ppos = Vector(graphicsRandom.randf(m_col.m_bbox.get_left()+8, m_col.m_bbox.get_right()-8), graphicsRandom.randf(m_col.m_bbox.get_top()+8, m_col.m_bbox.get_bottom()-8));
+  Vector pspeed = Vector(graphicsRandom.randf(speed.x-20, speed.x+20), graphicsRandom.randf(speed.y-20, speed.y+20));
+
+  // TODO: Rotate sprite rather than just use 2 different actions
+  // Approx. 1 particle per tile
+  if (graphicsRandom.randf(0.f, 100.f) < (m_col.m_bbox.get_width() / 32.f) * (m_col.m_bbox.get_height() / 32.f))
+  {
     // emit a particle
-    Vector ppos = Vector(graphicsRandom.randf(m_col.m_bbox.get_left()+8, m_col.m_bbox.get_right()-8), graphicsRandom.randf(m_col.m_bbox.get_top()+8, m_col.m_bbox.get_bottom()-8));
-    Vector pspeed = Vector(speed.x, speed.y);
-    Sector::get().add<Particles>(ppos, 44, 46, pspeed, Vector(0,0), 1, Color(.4f, .4f, .4f), 3, .1f,
-                                      LAYER_BACKGROUNDTILES+1);
+	  if (fancy_wind)
+    {
+	    Sector::get().add<SpriteParticle>("images/particles/wind.sprite", (abs(speed.x) > abs(speed.y)) ? "default" : "flip", ppos, ANCHOR_MIDDLE, pspeed, Vector(0, 0), LAYER_BACKGROUNDTILES+1); 
```

Comment:
I'm not sure if maybe you should move the `(abs(speed.x) > abs(speed.y)) ? "default" : "flip"` into its own variable. The line is quite long. Also, add spaces around the `+`

Yes, sorry for being mean and stricter than necessary, but that's just me being a good code reviewer. No hard feelings.